### DISCUSSION TITLE
RTSP cameras sync

### DISCUSSION THREAD

**Discussion by jimibit** [PyAV-Org/PyAV#1360] [1. Help]
Hi,

I’m working wit PyAv to stream 2 IP cameras (h264) connected to a Windows PC via a Poe switch.

I am remuxing the packets of each camera to mp4 segmented output files.

Each camera works with NTP (Network Time Protocol), so I can extract a time offset between each cam and the PC.

I have a buffer to save one GOP in order to avoid waiting for the next key_frame when a new remux starts, in separate threads I rework the videos by encoding and making new key_frames where needed to start.

I’m trying to sync the 2 videos: is there a way to extract the absolute cams timestamps from each packet ?

If no, any other suggestion for sync-ing ?

Thank you,

Jimi

**Comment by ncheng89** [comment]
I also have the same problem. Have you found a way to solve the synchronization problem?
